res/mushroom_complete.yaml

```diff
@@ -0,0 +1,53 @@
+type: horizontal-stack
+cards:
+  - type: vertical-stack
+    cards:
+      - type: horizontal-stack
+        cards:
+          - type: custom:mushroom-template-card
+            primary: >-
+              {{ 'Heures creuses' if
+              is_state('binary_sensor.rte_tempo_heures_creuses', 'on') else
+              'Heures pleines' }}
+            secondary: >-
+              {{ 'Tarif ' + states('sensor.rte_tempo_couleur_actuelle').lower() }}
+            icon: mdi:lightning-bolt
+            entity: sensor.rte_tempo_couleur_actuelle
+            icon_color: |-
+              {% if is_state('sensor.rte_tempo_couleur_actuelle', 'Blanc') %}
+                  white
+              {% elif is_state('sensor.rte_tempo_couleur_actuelle', 'Bleu') %}
+                  blue
+              {% elif is_state('sensor.rte_tempo_couleur_actuelle', 'Rouge') %}
+                  red
+              {% endif %}
+            badge_color: ""
+            badge_icon: ""
+            fill_container: false
+          - type: custom:mushroom-template-card
+            primary: Demain
+            secondary: "{{ states('sensor.rte_tempo_prochaine_couleur') }}"
+            icon: mdi:calendar
+            icon_color: |-
+              {% if is_state('sensor.rte_tempo_couleur_actuelle', 'Blanc') %}
```

Comment:
Thanks for the nice templates.
Typo here and line 34 and 36 below.
Replace `rte_tempo_couleur_actuelle` by `rte_tempo_prochaine_couleur`

Reply:
Indeed, sorry. Got same typo on `mushroom_colors.yaml`
Fixed both.